prometheus/prometheus.yaml

```diff
@@ -24,3 +24,10 @@ scrape_configs:
       - targets: [ "host.docker.internal:9093" ]
         labels:
           bot: "batcher"
+  
+  - job_name: "aligned-tracker"
+    scrape_interval: 1s
+    static_configs:
+      - targets: [ "host.docker.internal:4001" ]
```

Comment:
I think it is a bad idea to scrap to the same port where the API es served. It should another one